db/migrate/20170217085547_add_type_to_cloud_object_store_container.rb

```diff
@@ -0,0 +1,5 @@
+class AddTypeToCloudObjectStoreContainer < ActiveRecord::Migration[5.0]
+  def change
+    add_column :cloud_object_store_containers, :type, :string
```

Comment:
As discussed with @Fryguy https://github.com/ManageIQ/manageiq/pull/13965#discussion_r103509136, the combined index is not needed here.